lib/miq_automation_engine/models/miq_ae_yaml_import.rb

```diff
@@ -8,6 +8,10 @@ def initialize(domain, options)
     @domain_name = domain
     @options     = options
     @restore     = @options.fetch('restore', false)
+    tenant_id    = @options.fetch('tenant_id', Tenant.root_tenant.try(:id))
```

Comment:
@kbrock 
The import can be called from the command line via rake or it can be called directly from within our app.
When invoked via rake we would need to support tenant_id and fetch the corresponding tenant object.